frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailsTable.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import ExpandableString from "metabase/query_builder/components/ExpandableString";
+import { isID } from "metabase/lib/schema_metadata";
+import { TYPE, isa } from "metabase/lib/types";
+import { formatValue, formatColumn } from "metabase/lib/formatting";
+import cx from "classnames";
+import { t } from "ttag";
```

Comment:
Please sort imports according to our [style guide](https://www.metabase.com/docs/latest/developers-guide/frontend.html#javascript-conventions)

Reply:
thanks for pointing me to the style guide. very helpful!